src/app/admin/components/manage-blog/manage-blog.component.ts

```diff
@@ -21,6 +21,7 @@ export class ManageBlogComponent implements OnInit {
   submitted: boolean;
   appBlogDialog: boolean;
   blogForm: FormGroup;
+  public now: Date = new Date();
```

Comment:
why date is public? do you use it outside of this file??

Reply:
i have change it